docs/experiments/developing_an_experiment.md

```diff
@@ -0,0 +1,116 @@
+# Test Pilot Recommended Development Process for Experiments
+
+The following process has proven to be the most streamlined in the Test Pilot
+world.  You're not required to follow it, but if you do things will probably
+move more smoothly and it'll be easier to get help for your experiment because
+it's a familiar process.
+
+## Sprints
+We're fans of sprinting.  The standard two weeks is a good balance of feature
+development and shipping.
+
+## Communication
+Yay meetings. :)
+
+- Pick Slack or IRC.  Having a channel on each just makes it harder to decide
+  where to talk.  Now that contributors can join Slack, most experiments choose
```

Comment:
> Now that contributors can join Slack...

I haven't heard this before - is there a page somewhere we can point contributors at to get a sign up?

Reply:
It's a manual add via service-now right now.  They are trialing it.  Some info on https://mana.mozilla.org/wiki/display/CCT/Slack

Reply:
Might be splitting hairs, but this could be worth mentioning - contributors can't *join* but they can in some cases be *invited* on a case-by-case basis. So we'd have to connect with them through other channels like github issues and discourse threads.